flake-info/src/bin/flake-info.rs

```diff
@@ -251,6 +254,9 @@ async fn run_command(
             Ok((exports, ident))
         }
         Command::NixpkgsArchive { source, channel } => {
+            // TODO: if NIXOS_CHANNELS environment variable is present check
```

Comment:
@karantan @ysndr  This is where step number 7 should be implemented. I've also added the instructions